README.md

```diff
@@ -1,8 +1,32 @@
 # SymPy Paper
 
-[![Join the chat at https://gitter.im/asmeurer/sympy-paper](https://badges.gitter.im/asmeurer/sympy-paper.svg)](https://gitter.im/asmeurer/sympy-paper?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+This is the source repository for the paper:
+
+> SymPy Authors, "SymPy: Symbolic Computing in Python", 2016.
+
+This repository contains or links to all of the information needed to reproduce
+the results in the paper.
+
+The latest rendered version of the PDF can be viewed via the
+[ShareLaTeX](http://sharelatex.com) CI system:
 
 [![PDF Status](https://www.sharelatex.com/github/repos/asmeurer/sympy-paper/builds/latest/badge.svg)](https://www.sharelatex.com/github/repos/asmeurer/sympy-paper/builds/latest/output.pdf)
 
+# Contributing
+
+Join our chat room to discuss contributions:
+
+[![Join the chat at https://gitter.im/asmeurer/sympy-paper](https://badges.gitter.im/asmeurer/sympy-paper.svg)](https://gitter.im/asmeurer/sympy-paper?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
+# License
+
+The contents of this repository is licensed under the [Creative Commons
+Attribution 4.0 International
+License](http://creativecommons.org/licenses/by/4.0).
+
+[![CCBY](https://i.creativecommons.org/l/by/4.0/80x15.png)](http://creativecommons.org/licenses/by/4.0)
```

Comment:
Also @certik, as a point of clarification when I said, "pretty much unheard of" I was not referring to documentation, which is often released under a software license and is generally considered part of the software project.

Rather, I was consider other large bodies of prose knowledge, such as wikipedia / wikimedia and the arxiv. Both of these are licensed under a CC license and not under MIT or similar. See section 7 of https://wikimediafoundation.org/wiki/Terms_of_Use and http://arxiv.org/help/license 

In particular, I think this paper falls into more of the arxiv category than anything else.  Since arxiv distributes the latex source and not just the resultant pdf, these source documents typically also fall under their CC or similar licence. 

I think the dual licencing is fine, but I just wanted to explain my thought process :)